pom.xml

```diff
@@ -427,7 +427,7 @@
       <plugin>
         <groupId>org.jacoco</groupId>
         <artifactId>jacoco-maven-plugin</artifactId>
-        <version>0.8.5</version>
+        <version>0.8.4</version>
```

Comment:
Why this downgrade?

Reply:
I was seeing an `Error while creating report: null: EOFException`, so I thought recent upgrade could be a reason.

Reply:
Seems like I was wrong in suspecting upgrade could be the cause of the exception. It has started coming up again :confused: https://ci.fabric8.io/job/kubernetes-client-pullreq/2382/console